modules/manage/pages/manage-security/manage-users-and-roles.adoc

```diff
@@ -131,28 +115,425 @@ on [.ui]*travel-sample*, the view is as follows:
+
+For the detailed steps required to establish external authentication by means of Native LDAP (which is the recommended mechanism), see xref:manage:manage-security/configure-ldap.adoc[Configure LDAP].
+
+To add an externally authenticated user, on the *Add New User* dialog, select the *External* option:
+
+image::manage-security/externalUserRadioButton.png[,100,align=middle]
+
+This removes the password-related fields from the dialog, since they are not required for the creation of an externally authenticated user.
+The fields for the user's names remain, and can be used.
+
+The view provided by the dialog's right-hand panel can be toggled, by means of the *Roles* and *Groups* tabs, which appear above the panel:
+
+image::manage-security/rolesAndGroupsToggle.png[,140,align=middle]
+
+The *Roles* option can be used for externally authenticated users, just as it is used for locally authenticated users.
+The only difference is that no password need be specified for externally authenticated users.
+
+The *Groups* option can only be used when _Native LDAP_ has been established as the means of external authentication, and _LDAP group-support_ has been enabled.
+It provides a _group-mapping_, whereby a user-group defined on Couchbase Server is mapped to an LDAP group on an external server; and the user's LDAP-group membership can thus be used to assign the user the roles established for the Couchbase-Server user-group.
+See xref:learn:security/authentication-overview.adoc#native-ldap-support[Native LDAP], for an overview; and xref:manage:manage-security/configure-ldap.adoc[Configure LDAP], for configuration steps.
+
+[#add-an-externally-authenticated-user-individually]
+==== Add an Externally Authenticated User without Group-Mapping
+
+If the externally authenticated user is _not_ to be associated with an LDAP-group mapping, assign roles individually, using the *Roles* view, just as you would for a locally authenticated user.
+For example, add an externally authenticated user, with username `externalUserIndividual`, and the `ClusterAdmin` role:
+
+image::manage-security/addNewIndividualLDAPuser.png[,420,align=middle]
+
+Then, left-click on the *Add User* button.
+
+image::manage-security/addUserTab2.png[,120,align=middle]
+
+The *Users & Groups* panel now appears as follows:
+
+image::manage-security/externalUserIndividual.png[,720,align=middle]
+
+The new external user is listed, with their *auth domain* specified as `External`.
+
+[#add-an-externally-authenticated-user-by-group]
+==== Add an Externally Authenticated User with Group-Mapping
+
+If the externally authenticated user _is_ to be associated with an LDAP-group mapping, do not assign roles to the user individually.
+Instead, toggle the right-hand panel, by left-clicking on the *Groups* tab:
+
+image::manage-security/groupsTab.png[,140,align=middle]
+
+This displays the Couchbase-Server user-groups that have been mapped to LDAP groups:
```

Comment:
Non necessarily mapped. Couchbase groups can be used without ldap groups. Admin just needs to create a group and add a used to this group (what ldap groups give us is that this is no need to add user to groups manually, this information is taken from LDAP server in this case).

Reply:
I understand, and have changed this.